Cosmos_sdk_updater.sh

```
@@ -0,0 +1,175 @@
+# Use base images for C++ development
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)